projects/cluster/roles/cluster_deploy_nginx_server/tasks/main.yml

```diff
@@ -75,6 +82,15 @@
     oc expose svc/nginx --port=http
        -n "{{ cluster_deploy_nginx_server_namespace }}"
 
+- name: Capture HTTPS Route Config 
+  shell:
+    set -o pipefail;
+    oc create route passthrough nginx-secure
+       --service=nginx --port=https
+       -n "{{ cluster_deploy_nginx_server_namespace }}"
+       --dry-run=client -oyaml
+      | tee "{{ artifact_extra_logs_dir }}/src/route_nginx-secure.yaml"
```

Comment:
@drewrip , no need for a new task, 
in the task below you can pipe:
```
oc create route ...
   | tee ...
   | oc apply -f ...
```
alternatively, you keep this task, and below you just do this:
```
of apply -f "{{ artifact_extra_logs_dir }}/src/route_nginx-secure.yaml"
```